Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -561,6 +573,12 @@ void CodeViewWidget::OnContextMenu()
   for (auto* action : {symbol_rename_action, symbol_size_action, symbol_end_action})
     action->setEnabled(has_symbol);
 
+  for (auto* action : {copy_target_memory, show_target_memory})
+  {
+    action->setEnabled(Core::GetState() == Core::State::Paused &&
```

Comment:
The expression is evaluated multiple times. This should be moved outside the for loop and put into a constant.

Something like this or similar:
```c++
const bool is_valid_target_addr = Core::GetState() == Core::State::Paused && IsInstructionLoadStore(PowerPC::debug_interface.Disassemble(addr))
for (auto* action : {copy_target_memory, show_target_memory})
{
  action->setEnabled(is_valid_target_addr);
}
```